app/stack/new-database/route.js

```diff
@@ -1,6 +1,8 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  requireAuthentication: true,
```

Comment:
yes — I left them in as a reminder of sorts that `Ember.Route` has been reopened. I am on the fence about dropping them. If you think they are just extra verbosity I will drop them.